.github/workflows/main.yaml

```diff
@@ -25,6 +25,14 @@ jobs:
           distribution: 'adopt'
           cache: 'maven'
 
+      - name: Cache eclipse plugins
```

Comment:
These are Jenkins plugins, not Eclipse plugins.

Reply:
Yes I implemented a plugin cache to speed up testing during development in eclipse, I suppose that's where the lapsus is from. I fixed it.